### DISCUSSION TITLE
Pagination

### DISCUSSION THREAD

**Discussion by Ildest** [fremtind/jokul#3759] [Spørsmål]
Hi! Do you have a pagination component? I am not able to find it, but it might be named something else?

**Comment by fremtindjonas** [comment]
Hi, sorry for the late reply! We have a pagination component that is used in the table component, it's called "Table Pagenavigation". Does that work for what you will be using it for?

**Comment by Ildest** [comment]
Thank you, I will check it out and see if it covers what we need. I am writing documentation for our pagination component and looking at Jøkul and other design systems, to see if smart things have already been said :-).
